lib/miq_content_exporter.rb

```diff
@@ -0,0 +1,8 @@
+module MiqContentExporter
+  def self.to_hash(initial_hash, key, elements)
```

Comment:
I'm not a fan of the to_hash naming as that has a conventional meaning across Ruby. Perhaps `export_to_hash`?